config.yaml

```diff
@@ -50,6 +50,7 @@ cachecontrol.user: private,max-age=2
 # Enable development related features, e.g. token generation endpoint
 developer.mode.enabled: false
 log.level: info
+deployments.timeout: 30
```

Comment:
Could you add a comment here explaining what the timeout is used for and the unit of time (seconds)?

Reply:
'timeout' is very generic here.  Could you make it deployments.http.timeout or otherwise more explanatory?  (as well as @ebaron 's note re: comment)